### PR TITLE
ICU-21078 Adding missing copyright notice (sorry!)

### PR DESCRIPTION
**Pull request by hagbard**
- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21078
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

**Comment by markusicu** [comment]
Same as PR #1120 except for the fixed copyright year? I am curious why you didn't fix it in the earlier PR and commit-amend...?

**Comment by hagbard** [comment]
> Same as PR #1120 except for the fixed copyright year? I am curious why you didn't fix it in the earlier PR and commit-amend...?

Because github web UI + ICU team settings kinda suck.

If I make a quick CL like this in the web UI I am forced to get it absolutely 100% right first commit.
If I don't then I cannot modify it because that's 2 commits and the settings for ICU prohibit squashing in the UI. Then I have to use my repo on the desktop, pull, squash and push (and even then that's probably a new PR, since it's from my repo, not a branch on the main repo).

So basically it's either "get it right first time and take 30 seconds" or "waste 5 minutes". So (since these are by definition small CLs) it's always less effort to just try again on a new branch and abandon the old one.

**Comment by markusicu** [comment]
> > Same as PR #1120 except for the fixed copyright year? I am curious why you didn't fix it in the earlier PR and commit-amend...?
> 
> If I make a quick CL like this in the web UI I am forced to get it absolutely 100% right first commit.
> If I don't then I cannot modify it because that's 2 commits

Not necessary. You could make the fix, `git commit --amend` which replaces the commit without adding a second one (effectively squashing right in your local workspace with just one extra command-line option), and force-push that. Hence my question about "commit-amend". This is the easiest option.

> and the settings for ICU prohibit squashing in the UI.

There is a good reason for not allowing the "squash & merge" button. We want to maintain the cross-linking between commits and tickets, and so the reviewers and the CI bots need to be able to see the commit message. "Squash & merge" opens it back up to manual editing and does not give another chance to look & check. We did have some bad commit messages from that in the early GitHub days.

> Then I have to use my repo on the desktop, pull, squash and push

Which is not that hard. I have done a few of those. (No need to pull first; you have your branch locally anyway.)

> (and even then that's probably a new PR, since it's from my repo, not a branch on the main repo).

No, not a new PR. Just force-push, and the PR picks it up. It will even show the diff across the force-push. When you rebase you also have to force-push etc.

And Shane's handy squash tool linked from the ticket CI bot does it for you with a couple of clicks.

Please do read http://site.icu-project.org/repository/gitdev
